### ISSUE TITLE
10 point question not being 'clicked'

### ISSUE THREAD

**Issue by AnthonyBe** [zengfu94/MicrosoftRewards#41]
The old Bing Rewards script used to complete the 10 point question, and I *thought* this script also did, but now I'm not sure as I'm finding it regularly on the site when I visit it and thus manually clicking it.

Can you confirm whether this script does handle the 10 point question?

**Comment by zengfu94** [comment]
It used to but MS changed a few things. I'm working on it.

**Comment by AnthonyBe** [comment]
Just confirming that I've retested with commit 4244f38 but still seeing same issue.

**Comment by zengfu94** [comment]
Yeah I haven't pushed the big rebase yet.

**Comment by zengfu94** [comment]
Fixed :)

**Comment by AnthonyBe** [comment]
So if it is working I should see "<email>: extra offer <stuff>" appear in the script output? Does it attempt to do these first before desktop/mobile searches (I'm assuming yes)?

**Comment by zengfu94** [comment]
The exact output is email: performed offer: link

**Comment by AnthonyBe** [comment]
I'm still seeing 10 point questions available after running script.

Here's a sample of the script out. As you can see, there's no mention of it performing the offer.

`[17:07:07] accountremoved1@domainremoved: logging in
...
[17:07:23] accountremoved1@domainremoved: saved cookies for later
[17:07:25] accountremoved1@domainremoved: using saved cookies
[17:07:25] accountremoved1@domainremoved: current points: 7203
[17:07:28] accountremoved1@domainremoved: desktop left: 0
[17:07:28] accountremoved1@domainremoved: mobile left: 1
[17:07:29] accountremoved1@domainremoved: not all searches completed on purpose!
[17:07:29] accountremoved1@domainremoved: points earned: 0
[17:07:29] accountremoved1@domainremoved: total points: 7203
[17:07:29] accountremoved1@domainremoved: scripted completed at 05:07 PM`

Anything I can do to assist troubleshooting?

**Comment by zengfu94** [comment]
Can you please delete all saved cookies, this update changed quite a bit with them.

**Comment by AnthonyBe** [comment]
Deleted cookies.
Updated to latest commit (a8edded)
Still not taking 10 point offer.

```
Update complete, you're on the latest version.
[11:38:15] sleeping for 9 seconds
[11:38:15] sleeping for 15 seconds
[11:38:15] sleeping for 30 seconds
[11:38:24] accountremoved1@domainremoved: logging in
...
[11:38:35] accountremoved1@domainremoved: saved cookies for later
[11:38:36] accountremoved1@domainremoved: using saved cookies
[11:38:38] accountremoved1@domainremoved: current points: 16078
[11:38:40] accountremoved1@domainremoved: desktop left: 1
[11:38:40] accountremoved1@domainremoved: mobile left: 0
[11:38:41] accountremoved1@domainremoved: not all searches completed on purpose!
[11:38:43] accountremoved1@domainremoved: points earned: 0
[11:38:43] accountremoved1@domainremoved: total points: 16078
[11:38:43] accountremoved1@domainremoved: scripted completed at 11:38 AM
[11:38:43] -= Script elapsed time: 29 seconds =-
```

**Comment by zengfu94** [comment]
Can you hit view source on the dashboard and copy and paste all of it into text file for me.

**Comment by AnthonyBe** [comment]
https://pastebin.com/c6LiL2Ye

**Comment by AnthonyBe** [comment]
![image](https://user-images.githubusercontent.com/17214552/31580669-b3329cc2-b1a2-11e7-8fb6-996eeaab19fe.png)

**Comment by zengfu94** [comment]
Try latest :)

**Comment by AnthonyBe** [comment]
Updated to commit d0b9358 - no change :(

**Comment by AnthonyBe** [comment]
Updated to commit 278d7c4 - no change :(

**Comment by zengfu94** [comment]
Okay can confirm that its fixed with latest. Can't believe it took so long, ENAU does not have DailyOffers it has DailyOffer lol

**Comment by AnthonyBe** [comment]
LOL. It all came down to 's' (8a8a92c)? Will give it a try and confirm :)

**Comment by zengfu94** [comment]
Regex is picky! Keep submitting issues, yours are always fun to solve lol

**Comment by AnthonyBe** [comment]
Just shows how sloppy Microsoft are to be inconsistent across locales for the same text.
I worked there for 12 years and I used to pick up these sorts of issues all the time!

**Comment by zengfu94** [comment]
Dang lucky. I want to eventually get a job at a big company like MS

**Comment by wastingoctober** [comment]
@zengfu94 I wouldn't put that you wrote a bing bot script on your resume to Microsoft ;)

**Comment by AnthonyBe** [comment]
ha ha.. well it depends. Some good managers would appreciate the creativity and use of automation to solve a problem (i.e. complete a repetitive manual task). Others would just see 'doing the wrong thing'. :)

**Comment by zengfu94** [comment]
Trust me I'm not. Maker of bing bong cannot get hired at even the crappiest places.

**Comment by zengfu94** [comment]
That's why I didn't use my real name, poor guy unemployed

**Comment by wastingoctober** [comment]
Can he really not? Regardless of the ethics of it all, doesn't it still show coding prowess? I would think a company would care more about skills than anything.

**Comment by zengfu94** [comment]
Yeah it's BS. Just like I can actually code better than probably 75% of computer science graduates and I even haven't even gone to college. However they would get hired over me just because a piece of paper.

**Comment by wastingoctober** [comment]
I hear ya. I'm a SysAdmin, and I literally only got hired (for desktop support at the time) because they misread my resume and thought I had graduated. The guy who would become my future boss (and whose position I took over when he left less than a year later) actually asked in the interview when I graduated. I'm just glad that I at least now have 5 years of experience, and experience trumps everything.
